server_admin/topics/roles/user-role-mappings/default-roles.adoc

```diff
@@ -8,10 +8,4 @@ To specify default roles go to the `Roles` left menu item, and click the `Defaul
 .Default Roles
 image:{project_images}/default-roles.png[]
 
-As you can see from the screenshot, there are already a number of _default roles_ set up by default.
-
-
-
-
-
-
+As you can see from the screenshot, there are a number of _default roles_ already set up.
```

Comment:
How about "The following screens shows that some _default roles_ already exist."
By avoiding "there is," it becomes more concise and results in active voice.